documentation/release_notes.markdown

```diff
@@ -10,6 +10,16 @@ canonical: "/puppetserver/latest/release_notes.html"
 [puppetserver.conf]: ./config_file_puppetserver.markdown
 [product.conf]: ./config_file_product.markdown
 
+## Puppet Server 6.12.1
+
+Released 14 July 2020
+
+### Enhancements
+
+- Users who’ve started logging jolokia debug info will now have to enable a setting in the config file to get that back. New users that want to get debug info will have to add that setting, in additional updating logback.xml as they would have had to do before. Now, by default, attempting to access uncreated metrics will quietly continue rather than throw exceptions. The throwing exceptions behavior will return if enabling debug output. [TK-488](https://tickets.puppetlabs.com/browse/TK-488)
```

Comment:
That sounds terrible, who wrote it?   ( ;

Reply:
"Jolokia will no longer log at debug level by default, which avoids large stacktraces for missing metrics. In order to re-enable debug output, set `debug` to `true` in `metrics.conf`, in addition to configuring the logging to `debug` in `logback.xml`."

Reply:
💯